webanno-api-annotation/src/main/java/de/tudarmstadt/ukp/clarin/webanno/api/annotation/util/WebAnnoCasUtil.java

```diff
@@ -857,7 +838,7 @@ public static void setFeature(FeatureStructure aFS, AnnotationFeature aFeature,
                 aFS.setStringValue(feature, (String) aValue);
                 break;
             case CAS.TYPE_NAME_BOOLEAN:
-                aFS.setBooleanValue(feature, aValue != null ? (boolean) aValue : false);
+                aFS.setBooleanValue(feature, aValue != null && (boolean) aValue);
```

Comment:
The effect is the same, but more difficult to understand than the previous code which basically read as intended: "if the value is !null, then use the value, otherwise set to false". What you have now reads as "if value is !null and value" which is much more difficult to understand.